gips/data/modis/modis.py

```diff
@@ -250,17 +250,97 @@ def fetch(cls, asset, tile, date):
 
         return retrieved_filenames
 
+_tillage_product_types = ('ndti', 'crc', 'crcm', 'sti')
+
+_tp_descriptions = {
+    'ndti': 'Normalized Difference Tillage Index',
+    'crc':  'Crop Residue Cover (uses BLUE)',
+    'crcm': 'Crop Residue Cover, Modified (uses GREEN)',
+    'sti':  'Standard Tillage Index',
+}
+
+_tillage_product_dict = {
+    pt: {'bands': [pt], 'description': _tp_descriptions[pt],
+         'assets': ['MCD43A4'], 'sensor': 'MCD',
+         'startdate': datetime.date(2000, 2, 18), 'latency': 15}
+    for pt in _tillage_product_types}
+
+def pti_prep_data(refl, missing, bname):
+    """Ready given band for tillage product processing.
+
+    Pass in a reflectance GeoImage for MCD43A4, a no-data value,
+    and a list of band names (strings, eg 'blue'.  More info:
+        https://landsat.usgs.gov/what-are-band-designations-landsat-satellites
+        https://modis.gsfc.nasa.gov/about/specifications.php#1
+    Returns a numpy array for the given band.
+    """
+    name_to_num = {'blue': 10, 'green': 11, 'swir1': 12, 'swir2': 13}
```

Comment:
I believe that blue= MODIS instrument Band 3 = dataset layer index 9, and so on
https://lpdaac.usgs.gov/dataset_discovery/modis/modis_products_table/mcd43a4_v006

Reply:
seconded: https://modis.gsfc.nasa.gov/about/specifications.php